karbon/karbon_cluster_registration_service_test.go

```diff
@@ -1,23 +1,33 @@
 package karbon
 
 import (
+	"context"
 	"fmt"
 	"net/http"
 	"strings"
 	"testing"
+	"time"
 
 	"github.com/keploy/go-sdk/integrations/khttpclient"
 	"github.com/keploy/go-sdk/keploy"
 	"github.com/keploy/go-sdk/mock"
+	v3 "github.com/nutanix-cloud-native/prism-go-client/v3"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
+	prismgoclient "github.com/nutanix-cloud-native/prism-go-client"
 	"github.com/nutanix-cloud-native/prism-go-client/internal/testhelpers"
 )
 
 const (
-	noCategoryMappingErrorMsg = "No category mappings provided"
-	noUUIDErrorMsg            = "uuid in body is required"
+	noCategoryMappingErrorMsg  = "No category mappings provided"
+	noUUIDErrorMsg             = "uuid in body is required"
+	noTaskIDCreateRespErrorMsg = "Task ID was not include the create response."
+	timeout                    = 60 * time.Second
+	taskSucceed                = "SUCCEEDED"
+	taskFailed                 = "FAILED"
+	taskAborted                = "ABORTED"
+	taskSuspended              = "SUSPENDED"
```

Comment:
Consider prefixing unexported globals with `_` for readability. That way we know it's a global without having to jump to the declaration.
https://github.com/uber-go/guide/blob/master/style.md#prefix-unexported-globals-with-_

Reply:
thanks for the recommendation